pwa/static/themes/zwolle.css

```diff
@@ -0,0 +1,346 @@
+  --utrecht-alert-border-width: 1px;
+  --utrecht-alert-color: var(--web-app-color-black);
+  --utrecht-alert-padding-block-start: var(--web-app-size-lg);
+  --utrecht-alert-padding-block-end: var(--web-app-size-lg);
+  --utrecht-alert-padding-inline-start: var(--web-app-size-lg);
+  --utrecht-alert-padding-inline-end: var(--web-app-size-lg);
+  --utrecht-alert-content-margin-inline-start: var(--web-app-size-lg);
+  --utrecht-alert-content-margin-inline-end: var(--web-app-size-lg);
+  --utrecht-alert-icon-gap: 12px;
+  --utrecht-alert-info-background-color: rgba(23, 145, 212, 0.1);
+  --utrecht-alert-info-border-color: rgba(23, 145, 212, 0.25);
+  --utrecht-alert-icon-info-color: #00689e;
+
+  --utrecht-alert-warning-background-color: rgba(237, 181, 0, 0.1);
+  --utrecht-alert-warning-border-color: rgba(237, 181, 0, 0.25);
+  --utrecht-alert-icon-warning-color: #a65600;
+
+  --utrecht-alert-error-background-color: rgba(227, 97, 77, 0.1);
+  --utrecht-alert-error-border-color: rgba(227, 97, 77, 0.25);
+  --utrecht-alert-icon-error-color: var(--rotterdam-color-red-tint-02);
+
+  --utrecht-alert-ok-background-color: rgba(0, 129, 31, 0.1);
+  --utrecht-alert-ok-border-color: rgba(0, 129, 31, 0.25);
+  --utrecht-alert-icon-ok-color: var(--rotterdam-color-base-green);
+
+  /* @denhaag alert icon */
+  --utrecht-icon-size: 1.5em;
+
+  --utrecht-feedback-safe-fill-background-color: hsla(139, 81%, 35%, 1);
+  --utrecht-feedback-warning-fill-background-color: hsla(41, 100%, 43%, 1);
+  --utrecht-feedback-danger-fill-background-color: hsla(7, 60%, 52%, 1);
+  --utrecht-feedback-neutral-fill-background-color: hsla(201, 100%, 36%, 1);
+
+  --utrecht-button-group-inline-gap: var(--rotterdam-space-scale-3);
+  --utrecht-action-activate-cursor: pointer;
+
+  /* @gemeente-denhaag: Breadcrumb */
+  --denhaag-breadcrumb-color: #747474;
+  --denhaag-breadcrumb-current-color: var(--rotterdam-color-gray-tint-09);
+
+  --utrecht-color-sample-border-radius: 4px;
+  --utrecht-button-icon-gap: var(--web-app-size-xs);
+
+  /* @gemeente-utrecht: table */
+  --utrecht-table-border-width: 0;
+  --utrecht-table-row-hover-background-color: var(--rotterdam-color-base-gray);
+  --utrecht-table-row-hover-color: var(--rotterdam-color-gray-tint-11);
+  --utrecht-table-row-focus-background-color: var(--rotterdam-color-base-gray);
+  --utrecht-table-row-focus-color: var(--rotterdam-color-gray-tint-11);
+
+  --utrecht-table-cell-padding-block-start: inherit;
+  --utrecht-table-cell-padding-block-end: inherit;
+  --utrecht-table-cell-padding-inline-start: inherit;
+  --utrecht-table-cell-padding-inline-end: inherit;
+
+  /* @gemeente-utrecht: radio button */
+  --utrecht-radio-button-size: inherit;
+
+  /* @gemeente-utrecht: link */
+  --utrecht-link-visited-color: inherit;
+}
+
+.utrecht-link {
+  display: inline-flex;
+  align-items: flex-start;
+}
+
+/******* END OF THEME TEMPLATE ********/
+
+/*
+   * Theme-specific generics
+  */
+:root {
+  --zwolle-primary-blue: #476db7;
+}

```

Comment:
I would assume that there's a `--zwolle` token available for this, instead of making our own one and hardcoding the value?